configs/sample_yamls/serviceops/sample_service_config.yml

```diff
@@ -0,0 +1,172 @@
+#
+########################## types ########################
+#
+types:
+# QUOTA
+- name: ratelimit_by_service_by_consumer
+  quota_params:
+    dimensions:
+      service: STRING
+      consumer: STRING
+    maxAmount: 500
+    expiration: 1s
+- name: ratelimit_by_service_by_datacenter
```

Comment:
Type and Constructors both can have the same name if operator choose to do so. Not sure what would be a good name for a Type here. @mandarjog any ideas ?